src/locations/service/LocationsService.ts

```diff
@@ -74,7 +75,17 @@ export default class LocationsService extends RestService {
       this.mapResponseToPromise(requestUrl, deferred, error, response, body, this.EXPECTED_GET_OR_UPDATE_STATUS));
     return deferred.promise as any;
   }
-
+  public updateClosures(locationId: string, closedDates: LocationClosureRequest): Promise<LocationClosureResponse> {
+    const deferred = Q.defer<Location>();
+    const requestUrl = `/${Version.V1}${Resource.LOCATIONS}/${locationId}/closures`;
+    const options = {
+      data: closedDates
+    };
+    this.client.put(requestUrl, options, (error, response, body) => {
+      return this.mapResponseToPromise(requestUrl, deferred, error, response, body, this.EXPECTED_GET_OR_UPDATE_STATUS);

```

Comment:
@ManojKumarChannelApe Ensure we're correctly mapping the response after updating.  See the `mapLocationClosuresResponse()` method in `getClosures()`.
I caught this by seeing that the returned dates were still in the ISO string format, instead of JS date objects.